kotlin/display-map/src/main/res/drawable-v24/ic_launcher_foreground.xml

```diff
@@ -0,0 +1,34 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+        xmlns:aapt="http://schemas.android.com/aapt"
+        android:width="108dp"
+        android:height="108dp"
+        android:viewportHeight="108"
+        android:viewportWidth="108">
+  <path
```

Comment:
I don't know what these two launcher xml files do, but I assume they're intended (and important!) :)

Reply:
Adaptive drawables for the launcher icons - default for kotlin app template with latest AS.